App/Redux/MockBridge.ts

```diff
@@ -0,0 +1,39 @@
+import { createAction, getType, ActionType } from 'typesafe-actions'
```

Comment:
MockBridge is just creating methods that were otherwise missing or needed to call to pull app logic out of sdk functions.